python/dgl/contrib/graph_store.py

```diff
@@ -490,7 +560,12 @@ def init_ndata(self, ndata_name, shape, dtype):
             The data type of the node embedding. The currently supported data types
             are "float32" and "int32".
         """
-        self.proxy.init_ndata(ndata_name, shape, dtype)
+        init = self._node_frame.get_initializer(ndata_name)
+        if init is None:
+            self._node_frame._frame._warn_and_set_initializer()
+        init = self._node_frame.get_initializer(ndata_name)
+        init = self._init_manager.serialize(init)
+        self.proxy.init_ndata(init, ndata_name, shape, dtype)
         self._init_ndata(ndata_name, shape, dtype)
```

Comment:
It looks like the data is initialized twice? (once remotely, once locally?)

Reply:
the first init is to initialize the data in the remote server. the second one is actually to map data in the server to the local process. there is no initialization.